### ISSUE TITLE
FR in database is roughly twice the size of OA file

### ISSUE THREAD

**Issue by pnoll1** [pnoll1/osmand_map_creation#75]
**Describe the bug**
Database is nearly double what OA file is after one month

**To Reproduce**
Steps to reproduce the behavior, Mention an exact address if possible:
1. nvim fr/countrywide-addressess-country.geojson
2.  :$ and look for line count
3. psql -d gis -c 'select count(*) from fr_countrywide_addresses_country;

**Expected behavior**
OA file and database should roughly match with the database being bigger since it keeps all addresses it's ever seen

**Comment by pnoll1** [comment]
Looks like the hash changed even though content didn't
```
       6 | 141bf71b92dfee87 | 56     | Boucle du Closet |      | L'Abergement-de-Varey |          |        | 01640    | 01002_0050_00056 | 0101000020E6100000DF516342CCB51540FAEFC16B97004740
       5 | d588daaaca916d6b | 54     | Boucle du Closet |      | L'Abergement-de-Varey |          |        | 01640    | 01002_0050_00054 | 0101000020E6100000849ECDAACFB515402FFD4B5299004740
    1131 | cb95c36f08df6bf7 | 54     | Boucle du Closet |      | L'Abergement-de-Varey |          |        | 01640    | 01002_0050_00054 | 0101000020E61000007EE19524CFB51540A0A4C00298004740
       4 | 5273c58d78225e03 | 38     | Boucle du Closet |      | L'Abergement-de-Varey |          |        | 01640    | 01002_0050_00038 | 0101000020E610000056478E7406B615404D4A41B797004740
    1130 | 0650a977f46c8f3b | 38     | Boucle du Closet |      | L'Abergement-de-Varey |          |        | 01640    | 01002_0050_00038 | 0101000020E610000056478E7406B615404D4A41B797004740
    1132 | fe4f6043bf20fce7 | 56     | Boucle du Closet |      | L'Abergement-de-Varey |          |        | 01640    | 01002_0050_00056 | 0101000020E6100000DF516342CCB51540FAEFC16B97004740
```

**Comment by pnoll1** [comment]
fixed in f524e25d063f6f6c8d661fe3d4c88343da98042d by using postgres constraints to dedupe
